tests/functionnal/Glpi/Inventory/Inventory.php

```diff
@@ -1597,7 +1597,7 @@ public function testUpdateComputer()
             'LIMIT' => $nblogsnow,
             'OFFSET' => $this->nblogs,
         ]);
-        $this->integer(count($logs))->isIdenticalTo(1); //FIXME: should be 0
+        $this->integer(count($logs))->isIdenticalTo(29); //FIXME: should be 0
```

Comment:
What are these new log entries ?

Reply:
No idea, I've not checked; that's not the point of current PR

Reply:
we checked a little with @stonebuzz without success at the moment.
While i agree there is no obvious changes in this pr which should increase this number, this is very worrying.
Just after computer creation (we are just before update of the computer), we shouldn't have logs.
The initial fixme was for an entry like "Item was updated ..." due to time of execution.

The new one are very strange and seems to update some softwares (at computer creation !!)

Reply:
From  ```TU``` we import same computer  (```computer_3.json```) two times to check (from second import) we have no additional logs as the PC has not changed.

Locally, when y test this behavior, some  ```SoftwareVersion``` drop / add link to computer (```Installation```)

Problem seems to occur on the ```manufacturer``` key from ```getFullCompareKey``` (```handle``` function)

If I delete the ```$val->manufacturers_id``` key from  ```getFullCompareKey```  (is not the solution we agree)

```php
    protected function getFullCompareKey(\stdClass $val, bool $with_version = true): string
    {
        return $this->getCompareKey([
            sha1($val->name),
            $with_version ? strtolower($val->version) : '',
            strtolower($val->arch ?? ''),
            //$val->manufacturers_id,
            $val->entities_id,
            $val->is_recursive,
            $this->getOsForKey($val)
        ]);
    }
```

```TU``` returns only one log about ```Agent last contact update``` 

I notice that the SQL query (from ```handle``` step) now retrieves the manufacturer name instead of the ID.
So ```$key_w_version``` (from database data) is constructed with manufacturer name, but  ```$key_w_version``` (from inventory data) seems to be constructed with manufacturer ID

Reply:
I'll take a look